test/partisan_SUITE.erl

```diff
@@ -681,8 +709,24 @@ start(_Case, Config, Options) ->
 
     StartFun = fun({_Name, Node}) ->
                         %% Start partisan.
-                        {ok, _} = rpc:call(Node, application, ensure_all_started, [partisan])
-                   end,
+                        {ok, _} = rpc:call(Node, application, ensure_all_started, [partisan]),
+                        %% Start a dummy registered process that saves in the environment
+                        %% whatever message it gets, it will only do this *x* amount of times
+                        %% *x* being the number of nodes present in the cluster
+                        Pid = rpc:call(Node, erlang, spawn,
```

Comment:
it's in the [check_forward_message method](https://github.com/lasp-lang/partisan/pull/55/files#diff-0055ac29ce5da44a47db144edd2356caR882), where `Node` sends a `forward_message` to a random member's registered process `store_proc`